### PR TITLE
Jwu90

### PR DESCRIPTION
**Pull request by jwu90**
skeleton of home page, added linked home and recording page

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -17,7 +17,8 @@
             </intent-filter>
         </activity>
         <activity android:name=".Record" />
-        <activity android:name=".ViewRecording"></activity>
+        <activity android:name=".ViewRecording" />
+        <activity android:name=".activity_record"></activity>
```
Comment (jstnzl) [inline app/src/main/AndroidManifest.xml]: I dont think this should/needs to be here?

File: app/src/main/java/com/example/justi/markthat/Home.java [hunk 2]
```diff
@@ -10,4 +12,9 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_home);
     }
+
+    public void record(View view) {
```
Comment (jstnzl) [inline app/src/main/java/com/example/justi/markthat/Home.java]: Is this going to record or recording? If its going to record then name it like goToRecord

File: app/src/main/res/layout/activity_home.xml [hunk 3]
```diff
@@ -6,13 +6,157 @@
     android:layout_height="match_parent"
     tools:context=".Home">
```
Comment (jstnzl) [inline app/src/main/res/layout/activity_home.xml:8]: Not the right place for this comment but it applies to every component in this file.
Give each component a meaningful name it'll be much easier for whoever has to work on this file next.

Comment (jstnzl) [reply app/src/main/res/layout/activity_home.xml:8]: By convention I think its camelCase

**Review by jstnzl — CHANGES_REQUESTED** [review]

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (jstnzl) [reply app/src/main/AndroidManifest.xml]: the .activity_record one i mean

Comment (jwu90) [reply app/src/main/AndroidManifest.xml]: it was from a deleted activity i accidentally made. deleted

File: app/src/main/java/com/example/justi/markthat/Home.java (hunk 2 above)
Comment (jwu90) [reply app/src/main/java/com/example/justi/markthat/Home.java]: fixed

File: app/src/main/res/layout/activity_home.xml [hunk 4]
```diff
@@ -143,7 +143,7 @@
         android:layout_marginEnd="16dp"
         android:layout_marginBottom="27dp"
         android:clickable="true"
-        android:onClick="record"
+        android:onClick="goTorecord"
```
Comment (jstnzl) [inline app/src/main/res/layout/activity_home.xml:146]: nitpick- should be goToRecord

**Review by jstnzl — CHANGES_REQUESTED** [review]
Still needs the Folder_BG1 -> folder_bg1 change
Delete it and commit that change and then the next commit add the image back in with the correct capitalization.
If thats too tedious you'll find something quicker on google probably

As for constraints I thought you can set them when dragging/dropping?

Just fix the folder issue, then make a new branch like jwu-home-styling to work on constraints after merge

**Review by jstnzl — APPROVED** [review]

**Pull request merged**